build.gradle

```diff
@@ -58,6 +59,7 @@ apply from: file('gradle/validation/versions-props-sorted.gradle')
 apply from: file('gradle/validation/validate-source-patterns.gradle')
 apply from: file('gradle/validation/config-file-sanity.gradle')
 apply from: file('gradle/validation/rat-sources.gradle')
+apply from: file('gradle/validation/dependency-check.gradle')
```

Comment:
Perhaps change the name to "owasp-dependency-check" so that it's distinctly different from other dependency checks (there are some -- checksums, licenses, etc.)?